### PR TITLE
Add missing variable labels

### PR DESCRIPTION
**Pull request by ezraporter**
# Description
This PR adds variable labels for `form_complete_pct`, `event_name`, and `repeat_type` which were previously missing. Those columns will now get labels when `make_labelled()` is used.

### PR Checklist
Before submitting this PR, please check and verify below that the submission meets the below criteria:

- [x] New/revised functions have associated tests
- [x] New/revised functions that update downstream outputs have associated static testing files (`.RDS`) updated under `inst/testdata/create_test_data.R`
- [x] New/revised functions use appropriate naming conventions
- [x] New/revised functions don't repeat code
- [x] Code changes are less than **250** lines total
- [x] Issues linked to the PR using [GitHub's list of keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [x] The appropriate reviewer is assigned to the PR
- [x] The appropriate developers are assigned to the PR
- [x] Pre-release package version incremented using `usethis::use_version()`

# Code Review
This section to be used by the reviewer and developers during Code Review after PR submission

### Code Review Checklist

- [ ] I checked that new files follow naming conventions and are in the right place
- [ ] I checked that documentation is complete, clear, and without typos
- [ ] I added/edited comments to explain "why" not "how"
- [ ] I checked that all new variable and function names follow naming conventions
- [ ] I checked that new tests have been written for key business logic and/or bugs that this PR fixes
- [ ] I checked that new tests address important edge cases

### CODE REVIEW

File: R/labelled.R [hunk 1]
```diff
@@ -96,8 +97,10 @@ make_labelled <- function(supertbl, format_labels = NULL) {
 
   event_labels <- c(
     redcap_event = "Event Name",
+    event_name = "Event Description",
     redcap_arm = "Arm Name",
-    arm_name = "Arm Description"
+    arm_name = "Arm Description",
+    repeat_type = "Repetition Type"
```
Comment (rsh52) [inline R/labelled.R:103]: Have we ever used the phrase "Repetition Type" before? I might just opt for leaving it as "Repeat Type"

**Review by rsh52 — COMMENTED** [review]
Just one minor comment, otherwise nice catch and update.

File: R/labelled.R (hunk 1 above)
Comment (ezraporter) [reply R/labelled.R]: Changed in the last commit!

**Review by rsh52 — APPROVED** [review]
